libs/store-harvester/src/store/encrypted.ts

```
@@ -0,0 +1,77 @@
+        const userError = new Error(errorMessage);
+        userError.name = 'DatabaseLockError';
+        throw userError;
+      }
+
+      log.fatal(err, "Couldn't load config DB");
+      throw err;
+    }
+  }
+
+  private async initEncryptedDb(db: PouchDB.Database<Stored>, password?: string): Promise<void> {
+    await db.setPassword(password ?? "hF,835-/=Pw\\nr6r");
```

ESLint

[error] Strings must use singlequote.

(quotes)